Update zenlytic supported features

geometrikz opened the pull request:
Hi there,

I'm an engineer at Zenlytic, and I just wanted to update this document. We have support for file uploads, a python sandbox, as well as a traditional BI and dashboard GUI. We also support teams and slack integration for chat, and emails for scheduled reporting.

Our semantic layer is completely open-source and actively improved. https://github.com/Zenlytic/metrics_layer

Commit:
update zenlytic supported features

zurferr commented:
@geometrikz thank you for the information. Can you point me to any documentation regarding Teams, Slack and the python sandbox? I was not able to find that quickly when looking at your official page or docs.

Commit:
change zenlytic ordering

Commit:
remove email checkbox from zenlytic

geometrikz commented:
https://docs.zenlytic.com/docs/zenlytic_ui/zoe

@zurferr Thank you for the feedback, we realized that we haven't put up official docs for these features yet! I also noticed that the table was ordered by the number of features so I moved Zenlytic to the right order after the changes.

We just updated our docs to include these. Let me know if you need more information.
